src/main/java/qna/domain/Question.java

```diff
@@ -59,50 +59,49 @@ public boolean isOwner(User writer) {
 
     public void addAnswer(Answer answer) {
         answer.toQuestion(this);
+        answers.add(answer);
+    }
+    
+    public DeleteHistories delete(User loginUser) throws CannotDeleteException {
+        deleteValidation(loginUser);
+        this.deleted = true;
+        DeleteHistories deleteHistories = new DeleteHistories();
+        deleteHistories.add(DeleteHistory.of(ContentType.QUESTION, id, writer));
+        deleteHistories.add(answers.delete().getDeleteHistories());
```

Comment:
명령과 조회를 분리해주신 부분이 어떤 의도인지는 알 것 같지만, Question의 delete는 DeleteHistories를 반환하고 있네요 🤔 코드의 통일성이 필요해보여요 :)

```suggestion
        deleteHistories.add(answers.delete());
```

Reply:
이부분을 놓쳤네요! 수정하겠습니다! 👍🏻